bin/morgue.js

```diff
@@ -255,6 +256,62 @@ function coronerError(argv, config) {
   throw Error(argv._[1]);
 }
 
+function coronerProject(argv, config) {
+  abortIfNotLoggedIn(config);
+
+  let subcommand = argv._[1];
+
+  var coroner = coronerClientArgv(config, argv);
+  var bpg = coronerBpgSetup(coroner, argv);
+
+  if(!subcommand) {
+    errx("Invalid project command. Try 'morgue project create <your-project-name>'")
+  }
+
+  if(subcommand === 'create') {
+    let project = argv._[2];
+    if (!project) {
+      errx("Missing project name");
+    }
+
+    let validationRe = /^[a-zA-Z0-9-]+$/;
+    let validProjName = validationRe.test(project);
+    if(!validProjName) {
+      errx("Illegal name only use a-z, A-Z, 0-9, or \"-\"");
+    }
+
+    if(!config || !config.config) {
```

Comment:
small nit - you can put it before the regex. Regex executes some code - those are if statement available since the beginning.